airnow/misc.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import re
+
+
+def validate_zip_code(x):
```

Comment:
Think this func will be used elsewhere? If its only validating args, maybe we should rename this file to something like `validate_args` and include `valid_iso_8601` from my upcoming PR. Also, we could raise `argparse.ArgumentTypeError` and stick w the whole proper argparse approach.

Reply:
I was thinking we might use it again for the wrapper functions (stuff like `conditions_zip`, `forecast_zip`, etc.), which is why I put it in a separate file and used the more generic error types. Definitely agree with renaming the file to something more sensible, though.

Reply:
Moved to `validation.py`. I think maybe moving the date validation logic into there would be nice too. (side note: a whole package dedicated to validating stuff could be HUGE).

Reply:
Great idea. I'll rebase and move `valid_iso_8601`.